File: CPP/1. LinearSearch/LinearSearch.cpp
//CSE 075 08307

#include<iostream>

using namespace std;

int main(){

    int arr[] = {10, 20, 80, 30, 60, 50, 110, 100, 130, 170};

    int n = sizeof(arr)/sizeof(arr[0]);

    //Finding the Number
    int i,num,num2=NULL,inx;
    cout << "Input Your Number : ";
    cin >> num;
    for(i=0;i<n;i++){
            if(num == arr[i]){
                num2 = arr[i];
                inx = i;
                break;
            }
    }

    if(!num2){
        cout << "No Number like : "<<num;
    }else{
        cout << "Your Number is : "<<num2<<" And Index is : "<<inx+1;
    }

    return 0;
}

File: Python/1. LinearSearch/LinearSearch.py
# The Main Array
from asyncio.windows_events import NULL


arr = [10, 20, 80, 30, 60, 50, 110, 100, 130, 170]

number = input("Enter Your Number : ")
number = int(number)

n = len(arr)

for i in range(0,n):
    if(arr[i] == number):
        present = 1
        index = i
        break
    else:
        present = NULL

if(present == 1):
    print(f"Your Number Index is : {index + 1}")
else:
    print("Your Number Is not present Here")